frontend/src/pages/LogsPage/LogsPage.tsx

```diff
@@ -91,7 +91,7 @@ const LogsPageInner = ({ timeMode, logCursor, startDateDefault }: Props) => {
 	}
 
 	const handleLevelChange = (level: LogLevel) => {
-		setQuery(`level:${String(level).toLowerCase()}`)
+		setQuery(`${query} level:${String(level).toLowerCase()}`)
```

Comment:
Unrelated but `toLowerCase` is unnecessary:

https://github.com/highlight/highlight/blob/d4957a9e3bd0deb1dc45a00dfbc654463cdcff4f/backend/private-graph/graph/schema.graphqls#L296-L303

This code might have pre-dated when we finalized lowercasing everything.

Reply:
Just cleaned this up!